README.md

```diff
@@ -12,16 +12,11 @@
     1. or with https: `git clone https://github.com/HealthEconomicsHackathon/signpost.git`
     
 1. install [hugo](https://gohugo.io/getting-started/installing/)
-1. install blogdown from R:
```

Comment:
I'll update this documentation more fully as part of https://github.com/HealthEconomicsHackathon/SHEPRD/issues/7 and https://github.com/HealthEconomicsHackathon/SHEPRD/issues/6